sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/InstancesSamples.cs

```diff
@@ -59,7 +57,7 @@ public async Task RunSamplesAsync(TimeSeriesInsightsClient client)
                 }
                 else
                 {
-                    Console.WriteLine($"Failed to create a Time Series Insights instance with Id '{tsiId}'.");
+                    Console.WriteLine($"Failed to create a Time Series Insights instance with Id '{tsiId}', Error Message: '{createInstanceErrors.Value[i].Message}.");
```

Comment:
It would probably be useful to show the user that other error details exist. I will make the change on your branch.

Reply:
I can add error code. Yes other details don't make much sense for the sample.